packages/flutter_tools/lib/src/base/logger.dart

```diff
@@ -337,15 +345,31 @@ class VerboseLogger extends Logger {
   @override
   Status startProgress(
     String message, {
+    @required Duration timeout,
     String progressId,
-    bool expectSlowOperation,
-    bool multilineOutput,
-    int progressIndicatorPadding,
+    bool multilineOutput = false,
+    int progressIndicatorPadding = kDefaultStatusPadding,
   }) {
+    assert(timeout != null);
+    assert(progressIndicatorPadding != null);
     printStatus(message);
-    return Status(onFinish: () {
-      printTrace('$message (completed)');
-    })..start();
+    final Stopwatch timer = Stopwatch()..start();
+    return SilentStatus(
+      timeout: timeout,
+      onFinish: () {
+        String time;
+        if (timeout > fastOperation) {
```

Comment:
You can use the protected `elapsedTime` property here

Reply:
it's in a different class